styles.css

```diff
@@ -113,13 +118,24 @@ body {
   display: flex;
   flex: 1;
   flex-direction: column;
-}
```

Comment:
Might be the cause of some of the issues, not sure though

Reply:
good catch Taylor